src/main/kotlin/com/github/seliba/devcordbot/command/impl/CommandClientImpl.kt

```diff
@@ -70,6 +72,21 @@ class CommandClientImpl(
     override val commandAssociations: MutableMap<String, AbstractCommand> = mutableMapOf()
     override val errorHandler: ErrorHandler = if (bot.debugMode) DebugErrorHandler() else HastebinErrorHandler()
 
+    /**
+     * Listens for new private messages
+     */
+    @EventSubscriber
+    fun onPrivateMessage(event: PrivateMessageReceivedEvent): Unit = dispatchPrivateMessageCommand(event.message)
+
+    private fun dispatchPrivateMessageCommand(message: Message) {
+        if (!bot.isInitialized) return
+
+        val author = message.author
```

Comment:
bot ignores commands from non-devcord guilds, probably also makes sense to ignore commands from non-devcord users?